### ISSUE TITLE
Are `pdflatex` and `doxygen` dependencies to be listed?

### ISSUE THREAD

**Issue by fluca1978** [agroal/pgagroal#480]
While building a new (FreeBSD) machine, I found that the building phase required `pdflatex`, but it seems this dependency is not listed in the `README`. Is it worth mentioning that there is the need for LaTeX stuff? I'm not sure if `python3-docutils` is pulling this on some systems.

Same happens for `doxygen`.

**Comment by jesperpedersen** [comment]
Yes, we should port https://github.com/pgmoneta/pgmoneta/blob/main/doc/DISTRIBUTIONS.md over, and add `docutils`, `pdflatex` and `doxygen` to the FreeBSD section

**Comment by decarv** [comment]
I got this message while building in a new Rocky machine:

```-- pdflatex needed. The generation process will be skipped.
CMake Error at /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find Doxygen (missing: DOXYGEN_EXECUTABLE dot)
[...]
 ```
 
 Should Doxygen required for a build?
